### PR TITLE
Private Cloud Release Notes - 2 June 2022

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ For information on the current status of deployment to Mendix for Private Cloud
 
 ## 2022
 
+### June 2nd, 2022
+
+#### Portal Improvements
+
+* We now provide the `mxpc-cli` configuration tool in versions which run on ARM64 architectures.
```
Comment (Nidhi251289) [inline content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md]: the ARM version is only available from operator version 2.5.1 onwards

**Review by Nidhi251289 — COMMENTED** [review]
I have provided my comment. the ARM version is only available from operator version 2.5.1 onwards

File: content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md]: Updated.

**Comment by MarkvanMents** [comment]
Release successful - merging.

**Pull request merged**